tests/test_globs_exact_match.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from bossman.plugins.akamai.property import PropertyResource
+from bossman import Bossman
+
+
+@pytest.fixture(scope="module")
+def resources():
+    resources = list()
+    resources.append(PropertyResource("deployment-artifacts/globalmaileurope/akamai/property/globalmaileurope.dhl.com", name="globalmaileurope.dhl.com"))
```

Comment:
@berk76 are you comfortable with dhl references remaining in the code?

Reply:
Hello @ynohat 

you are right - I made all suggested changes.

* __get_resources methods into functions: I have no strong opinion about it both variants seem ok to me
*  single __get_resources: yes, it makes sense - now it is simpler
* dhl references: I renamed them - thank you for note

Have a nice Christmas :-)